include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -18,6 +18,12 @@
 
 #include <deal.II/base/config.h>
 
+#include "deal.II/base/types.h"
+
+#include "deal.II/distributed/p4est_wrappers.h"
```

Comment:
Do you really need this here?

Reply:
No I don't. At some point VSCode suggested `internal::p4est::types` for something and must have included the header.